sdk/cmake/msvc.cmake

```diff
@@ -447,7 +451,7 @@ function(CreateBootSectorTarget _target_name _asm_file _binary_file _base_addres
         COMMAND ${CMAKE_C_COMPILER} /nologo ${_no_std_includes_flag} /I${REACTOS_SOURCE_DIR}/sdk/include/asm /I${REACTOS_BINARY_DIR}/sdk/include/asm ${_includes} ${_defines} /D__ASM__ /D_USE_ML /EP /c ${_asm_file} > ${_temp_file}
         DEPENDS ${_asm_file})
 
-    set(_asm16_command ${CMAKE_ASM16_COMPILER} /nologo /Cp /Fo${_object_file} /c /Ta ${_temp_file})
+    set(_asm16_command ${CMAKE_ASM16_COMPILER} /nologo /quiet /Cp /Fo${_object_file} /c /Ta ${_temp_file})
```

Comment:
Should the "/quiet" flag here also be specified with the version-conditional `${_quiet_flag}` ?

Reply:
yes